apps/damap-frontend/src/app/components/layout/layout.component.html

```diff
@@ -93,6 +70,15 @@
         <!-- University Logo End -->
       </mat-toolbar-row>
       <mat-toolbar-row class="mat-toolbar-row-grey">
+        <button
+          mat-icon-button
+          title="{{ 'layout.menu.mobile.close' | translate }}"
+          class="sidenav-btn"
+          (click)="toggleSidenav()">
+          <mat-icon class="material-icon-on-secondary-container">
+            {{ sidenav.opened ? "close" : "menu" }}
+          </mat-icon>
+        </button>
```

Comment:
Can we move the button to the left side of the header? I think it provides a better separation of real content and navigation item

Reply:
With the new design, it would also provide a better separation of content and navigation, when the menu button is in the header. I guess it is an issue that the header right now is not always visible.

The old angular docs site also has this kind of style with the menu button in the sticky header: https://v17.angular.io/docs

Reply:
The `x` icon in general might be confusing (could mean delete, remove, close). Maybe we always show the `menu` icon so the meaning is clear?

Reply:
@DerekMolnar

Reply:
@lpandath @rekt-hard thank you for working through this.  I do think the x icon could be confusing.   Doesn't angular have an sidebar specific icons?  Examples: https://www.flaticon.com/free-icons/sidebar